api/src/opentrons/protocol_api/core/engine/module_core.py

```diff
@@ -696,18 +712,35 @@ def is_lid_on(self) -> bool:
         return abs_state.is_lid_on
 
 
-class FlexStackerCore(ModuleCore, AbstractFlexStackerCore):
+@dataclass
+class _CoreTrio:
```

Comment:
```suggestion
class _CoreStackerTrio:
```
I think this needs to be related to the stacker?

Reply:
sure. it will only ever be used within this module which is why I was loose with the naming

Reply:
oh Sorry I missed that its within the `FlexStackerCore`! you can leave as is then!